src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx

```diff
@@ -63,12 +63,28 @@ class CreateKnownSchemeProposal extends React.Component<IProps, IState> {
     const actions = props.genericSchemeInfo.actions();
     this.state = {
       actions,
-      currentAction:  actions[0],
+      currentAction: actions[0],
       tags: new Array<string>(),
     };
   }
+  
+  private async getBountyEth(values: IFormValues): Promise<string> {
+    const currentAction = this.state.currentAction;
+    let ethToSend = 0;
+
+    for (const field of currentAction.getFields()) {
+      if (field.name === "_depositAmount" || field.name === "_amount" && this.props.genericSchemeInfo.specs.name === "Standard Bounties") {
```

Comment:
This is still not good, you should add some more brackets to bet the correct logic - but please see the comment below.

Reply:
8fa6cde